### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by staskostenkov**
13.07.2020

### CODE REVIEW

File: site/ru/guide/customizing_what_happens_in_fit.ipynb [hunk 1]
```diff
@@ -0,0 +1,628 @@
...
+   "metadata": {
+    "colab_type": "text",
+    "id": "vApNeEfvLLc4"
+   },
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://www.tensorflow.org/guide/keras/customizing_what_happens_in_fit\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on TensorFlow.org</a>\n",
+    "  </td>\n",
+    "  <td>\n",
```
Comment (lamberta) [inline site/ru/guide/customizing_what_happens_in_fit.ipynb:58]: This button URLs are incorrect, see the nblint results. Should point to this notebook in the docs-l10n repo

File: site/ru/guide/keras/customizing_what_happens_in_fit.ipynb [hunk 2]
```diff
@@ -0,0 +1,620 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "tE4yrX22rlL4"
+   },
+   "source": [
+    "## Предоставление собственного шага оценки\n",
+    "\n",
+    "Надо сделать тоже самое для `model.evaluate()`? Тогда надо таким же способом \n",
```
Comment (stabuev) [inline site/ru/guide/keras/customizing_what_happens_in_fit.ipynb:372]: Надо сделать тоже самое для `model.evaluate()`? -> Что если Вы хотите сделать то же самое для вызовов `model.evaluate()`?

Может так перефразировать?

File: site/ru/guide/keras/customizing_what_happens_in_fit.ipynb [hunk 3]
```diff
@@ -0,0 +1,620 @@
...
+   },
+   "outputs": [],
+   "source": [
+    "class CustomModel(keras.Model):\n",
+    "    def test_step(self, data):\n",
+    "        # Распаковка данных\n",
+    "        x, y = data\n",
+    "        # Вычисление предсказаний\n",
+    "        y_pred = self(x, training=False)\n",
+    "        # Обновление метрики отслеживания потерь\n",
```
Comment (stabuev) [inline site/ru/guide/keras/customizing_what_happens_in_fit.ipynb:391]: метрики отслеживания потерь -> метрик отслеживающих потери

мне кажется по смыслу так лучше

File: site/ru/guide/keras/customizing_what_happens_in_fit.ipynb [hunk 4]
```diff
@@ -0,0 +1,620 @@
...
+    "        self.compiled_loss(y, y_pred, regularization_losses=self.losses)\n",
+    "        # Обновление метрик.\n",
+    "        self.compiled_metrics.update_state(y, y_pred)\n",
+    "        # Возвращение словаря, отображающий имена метрик к текущему значению.\n",
+    "        # Заметьте, что это будет включать потерю (отслеживается в self.metrics).\n",
```
Comment (stabuev) [inline site/ru/guide/keras/customizing_what_happens_in_fit.ipynb:396]: потерю -> потери

File: site/ru/guide/keras/customizing_what_happens_in_fit.ipynb [hunk 5]
```diff
@@ -0,0 +1,620 @@
...
+    "## Подведение итогов: пример GAN от начала до конца\n",
+    "\n",
+    "Давайте рассмотрим сквозной пример, который использует все, что вы только что узнали.\n",
+    "\n",
+    "Давайте рассмотрим:\n",
+    "\n",
+    "- Сеть генератора для генерации изображений размером 28x28x1.\n",
+    "- Сеть дискриминаторов, предназначенная для классификации изображений 28x28x1 на два класса («фейковые» и \"реальные\").\n",
+    "- Один оптимизатор для каждого.\n",
+    "- Функция потери для обучения дискриминатора.\n",
```
Comment (stabuev) [inline site/ru/guide/keras/customizing_what_happens_in_fit.ipynb:428]: потери -> потерь

File: site/ru/guide/keras/customizing_what_happens_in_fit.ipynb [hunk 6]
```diff
@@ -0,0 +1,620 @@
...
+    "\n",
+    "- A generator network meant to generate 28x28x1 images.\n",
+    "- A discriminator network meant to classify 28x28x1 images into two classes (\"fake\" and\n",
+    "\"real\").\n",
+    "- One optimizer for each.\n",
+    "- A loss function to train the discriminator.\n"
```
Comment (stabuev) [inline site/ru/guide/keras/customizing_what_happens_in_fit.ipynb:434]: Кусок на удаление :)

File: site/ru/guide/keras/customizing_what_happens_in_fit.ipynb [hunk 7]
```diff
@@ -0,0 +1,620 @@
...
+   "execution_count": 0,
+   "metadata": {
+    "colab_type": "code",
+    "id": "xiE4ZsCtjI9B"
+   },
+   "outputs": [],
+   "source": [
+    "from tensorflow.keras import layers\n",
+    "\n",
+    "# Созданиеscriminator = keras.Sequential(\n",
```
Comment (stabuev) [inline site/ru/guide/keras/customizing_what_happens_in_fit.ipynb:448]: тут сбился код, нужно поправить

File: site/ru/guide/keras/customizing_what_happens_in_fit.ipynb [hunk 8]
```diff
@@ -0,0 +1,620 @@
...
+    "        layers.LeakyReLU(alpha=0.2),\n",
+    "        layers.Conv2D(128, (3, 3), strides=(2, 2), padding=\"same\"),\n",
+    "        layers.LeakyReLU(alpha=0.2),\n",
+    "        layers.GlobalMaxPooling2D(),\n",
+    "        layers.Dense(1),\n",
+    "    ],\n",
+    "    name=\"discriminator\",\n",
+    ")\n",
+    "\n",
+    "# Create the generaСоздание_dim = 128\n",
```
Comment (stabuev) [inline site/ru/guide/keras/customizing_what_happens_in_fit.ipynb:461]: тут тоже сбился код, нужно подправить по оригиналу

File: site/ru/guide/keras/customizing_what_happens_in_fit.ipynb [hunk 9]
```diff
@@ -0,0 +1,620 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "jyyxuepxgMuF"
+   },
+   "source": [
+    "Вот полнофункциональный класс GAN, переопределяющий `compile ()` для использования собственной сигнатуры, и реализующий весь алгоритм GAN в 17 строках в `train_step`:"
```
Comment (stabuev) [inline site/ru/guide/keras/customizing_what_happens_in_fit.ipynb:486]: `compile ()` -> `compile()`

File: site/ru/guide/keras/customizing_what_happens_in_fit.ipynb [hunk 10]
```diff
@@ -0,0 +1,620 @@
...
+    "        generated_images = self.generator(random_latent_vectors)\n",
+    "\n",
+    "        # Соединение фейковых и реальных изображений\n",
+    "        combined_images = tf.concat([generated_images, real_images], axis=0)\n",
+    "\n",
+    "        # Сборка ярлыков, отличающих реальные изображения от поддельных\n",
+    "        labels = tf.concat(\n",
+    "            [tf.ones((batch_size, 1)), tf.zeros((batch_size, 1))], axis=0\n",
+    "        )\n",
+    "        # Добавление случайного шума к ярлыкам - выжный трюк!\n",
```
Comment (stabuev) [inline site/ru/guide/keras/customizing_what_happens_in_fit.ipynb:528]: выжный -> важный

File: site/ru/guide/keras/customizing_what_happens_in_fit.ipynb [hunk 11]
```diff
@@ -0,0 +1,620 @@
...
+    "            d_loss = self.loss_fn(labels, predictions)\n",
+    "        grads = tape.gradient(d_loss, self.discriminator.trainable_weights)\n",
+    "        self.d_optimizer.apply_gradients(\n",
+    "            zip(grads, self.discriminator.trainable_weights)\n",
+    "        )\n",
+    "\n",
+    "        # Выборка случайных точек в скрытом пространстве\n",
+    "        random_latent_vectors = tf.random.normal(shape=(batch_size, self.latent_dim))\n",
+    "\n",
+    "        # Сборка ярлыков которые показывают \"все реальные изображения      misleading_labels = tf.zeros((batch_size, 1))\n",
```
Comment (stabuev) [inline site/ru/guide/keras/customizing_what_happens_in_fit.ipynb:543]: Сбился код, нужно сверить с оригиналом и подправить

File: site/ru/guide/keras/customizing_what_happens_in_fit.ipynb [hunk 12]
```diff
@@ -0,0 +1,620 @@
...
+    "\n",
+    "        # TraТренировкаnerator (notзаметьте что otнеpdобновляются веса в discriminator)!      # oh tf.GradientTape() as tape:\n",
```
Comment (stabuev) [inline site/ru/guide/keras/customizing_what_happens_in_fit.ipynb:545]: Тут тоже каша

**Comment by lamberta** [comment]
Thanks, @staskostenkov 
I've manually verified the CLA. I think the bot is having difficulty because some of the commit email address don't match the one provided for the CLA. That should hopefully work itself out in future pull requests.

But looks like the CI is failing. Need to update the `class=tfo-notebook-buttons` URLs to point to this translated notebook (not the source notebook). Click the "Details" link in the failing CI tests to see the console output.

Please format notebooks with [nbfmt](https://github.com/tensorflow/docs/blob/master/tools/tensorflow_docs/tools/nbfmt/__main__.py) and use [nblint](https://github.com/tensorflow/docs/blob/master/tools/tensorflow_docs/tools/nblint/__main__.py) to check style. Thank you

```
$ python3 -m pip install -U --user git+https://github.com/tensorflow/docs

$ python3 -m tensorflow_docs.tools.nbfmt notebook.ipynb
$ python3 -m tensorflow_docs.tools.nblint --arg=repo:tensorflow/docs-1l0n \
    --styles=tensorflow,tensorflow_docs_l10n notebook.ipynb
```

**Comment by lamberta** [comment]
Oh, I'd also recommend trying the GitLocalize interface for Russian translations: https://gitlocalize.com/tensorflow/docs-l10n/
This should hopefully be much easier for contributors.

**Comment by lamberta** [comment]
Thank you for the contribution but this PR seems to have stalled. Closing, for now.

As mentioned on the [mailing list](https://groups.google.com/a/tensorflow.org/g/docs/c/G23C6ZxjiUI/m/z8G87HTfBQAJ), we're directing the community to work in our [GitLocalize project](https://gitlocalize.com/tensorflow/docs-l10n). The community can submit pull requests through GitLocalize and provide reviews.

This file can be found here: https://gitlocalize.com/repo/4592/ru/site/en-snapshot/guide/keras/customizing_what_happens_in_fit.ipynb

**Pull request closed without merging**